src/include/migraphx/op/pointwise.hpp

```diff
@@ -41,6 +41,8 @@ struct pointwise
 
     shape compute_shape(const std::vector<shape>& inputs, std::vector<module_ref> mods) const
     {
+        assert(inputs.size() > 0);
```

Comment:
this should be using `check_shapes().has_atleast(1)`.

Reply:
On it

Reply:
done